eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -134,6 +134,10 @@ the main ServiceBusClientBuilder. -->
             files="com.azure.storage.blob.BlobAsyncClient.java"/>
   <suppress checks="com.azure.tools.checkstyle.checks.ServiceClientCheck"
             files="com.azure.storage.blob.BlobClient.java"/>
+  <!-- Suppress client mutability check since LeaseClients need to update a changing lease ID -->
+  <suppress checks="com.azure.tools.checkstyle.checks.ServiceClientCheck"
+            files="com.azure.storage.blob.specialized.BlobLeaseAsyncClient.java"
+            lines="67"/>
```

Comment:
exact line might be painful to maintain. I'd rather follow examples above and apply to whole file.